Code/Tools/FBuild/FBuildCore/BFF/Functions/FunctionPrint.cpp

```diff
@@ -124,7 +171,7 @@ FunctionPrint::FunctionPrint()
 
 // PrintVarRecurse
 //------------------------------------------------------------------------------
-/*static*/ void FunctionPrint::PrintVarRecurse( const BFFVariable & var, uint32_t indent )
+/*static*/ void FunctionPrint::PrintVarRecurse( const BFFVariable & var, uint32_t indent, uint32_t maxIndent )
```

Comment:
I went with `maxIndent` instead of a boolean because I thought it might be useful to specify a struct would only expand one level deep (if `maxIdent == indent+1`), or to control struct depth + stack depth to keep the output from getting too "wide".  I only ended up using it as a boolean, though, and didn't hook up any options that would control width.